src/server/services/helpers/getDuplicates.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function getDuplicates(cleanedArray) {
+    let duplicatesObject = {};
+    duplicatesObject[cleanedArray[0][0]] = cleanedArray[0][1];
+
+    for (let i = 1; i <= (cleanedArray.length - 1); i++) {
```

Comment:
better to use `forEach` instead of `for`

Reply:
Why forEach is better? There is some info, that for works faster. Just a question, I am not sure